aroma/tests/conftest.py

```diff
@@ -23,7 +26,54 @@ def testpath(tmp_path_factory):
     return tmp_path_factory.getbasetemp()
 
 
+@pytest.fixture
+def featurespath():
+    return op.join(op.dirname(aroma.__file__), 'tests', 'data', 'features_test')
+
+
 @pytest.fixture
 def nilearn_data(testpath):
-    return fetch_development_fmri(n_subjects=1, age_group="adult",
-                                  data_dir=str(testpath))
```

Comment:
Why would you eliminate the ``data_dir=str(testpath)``, are you sure this doesn't make that when executing pytest, non-temporally files are created

Reply:
Why would you leave ``testpath`` as an unused argument

Reply:
Good catch! I did not mean to remove that.